src/analyzer/protocol/modbus/modbus-analyzer.pac

```diff
@@ -81,13 +81,20 @@ refine connection ModbusTCP_Conn += {
 		%{
 		return confirmed && orig_pdu && resp_pdu;
 		%}
+
+	function IsValidHeader( header: ModbusTCP_TransportHeader ) : bool
+        %{
+        // Protocol Identifier must be 0 as per specs
+        // Length of Modbus APDU must be at least 2 (Unit Identifier and Function Code)
+        return ${header.pid} == 0 && ${header.len} >= 2;
+        %}
```

Comment:
Nit: The rest of the file uses tabs, but the new function bodies use spaces. Could they be swapped to tabs to be consistent?